tasks.py

```diff
@@ -181,11 +185,15 @@ def help(c):
                 host = re.search('Host\(\`(?P<host>.*)\`\)', router['rule']).group('host')
                 if host:
                     scheme = 'https' if 'https' in router['using'] else router['using'][0]
-                    print(Fore.YELLOW + "* " + scheme + "://" + host)
+                    print("* " + Fore.YELLOW + scheme + "://" + host + Fore.RESET)
+        print('')
     except:
         pass
 
 
+    print('Run ' + Fore.GREEN + 'inv -l' + Fore.RESET + ' to list all the available tasks.')
+    c.run('inv --list')
```

Comment:
Wouldn't that list "hide" the urls at the end of `fab start`?

Reply:
We could put the list before the URL, WDYT ?

Reply:
:+1: